rust/README.md

```diff
@@ -110,6 +110,9 @@ PR be sure to run the following and check for lint issues:
 cargo +stable fmt --all -- --check
 ```
 
+We can use "git pre-commit hook" to automate this process: copy or soft link [pre-commit.sh](pre-commit.sh)
+as file `.git/hooks/pre-commit`.
```

Comment:
Just to make it dead simple to people less experienced in git, would it be possible to paste the command here? E.g. 

```bash
cp rust/pre-commit.sh ../.git/hooks/pre-commit
```

or 

```bash
ln ...
```

Reply:
@jorgecarleitao good idea!

I have tried this way but finally aborted.
One of the reason is: one line command for copy/paste is not enough -- if the pre-commit file already exists, copy/link may fail or override them unexpectedly, thus additional check/prompt would make the commands lengthy. 

Anyway, let me try again.